<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>610</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Pristina</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <italic>false</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>tbPrivateDiary</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="leftLabel">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Lucida Sans Unicode</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Dear Diary:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="textEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="rightLabel">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans Unicode</family>
    </font>
   </property>
   <widget class="QMenu" name="menuDiary">
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Diary</string>
    </property>
    <addaction name="actionNew_Diary_Entry"/>
    <addaction name="actionOpen_Diary_Entry"/>
    <addaction name="actionSave_Diary_Entry"/>
    <addaction name="actionFont_Diary_Entry"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDiary"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Diary_Entry">
   <property name="text">
    <string>New Entry</string>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans Unicode</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionOpen_Diary_Entry">
   <property name="text">
    <string>Open Entry</string>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans Unicode</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionSave_Diary_Entry">
   <property name="text">
    <string>Save Entry</string>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans Unicode</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionFont_Diary_Entry">
   <property name="text">
    <string>Font</string>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans Unicode</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
